src/test/java/com/emailcrawler/crawlerutil/EmailCrawlerTest.java

```diff
@@ -0,0 +1,41 @@
+package com.emailcrawler.crawlerutil;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+import java.util.List;
+
+import com.emailcrawler.crawlerutil.EmailCrawlerTest;
+
+/**
+ * Unit test for simple App.
+ */
+public class EmailCrawlerTest extends TestCase {
+	/**
+	 * Create the test case
+	 *
+	 * @param testName
+	 *            name of the test case
+	 */
+	public EmailCrawlerTest(String testName) {
+		super(testName);
+	}
+
+	/**
+	 * @return the suite of tests being tested
+	 */
+	public static Test suite() {
+		return new TestSuite(EmailCrawlerTest.class);
+	}
+
+	/**
+	 * Sample Test
+	 */
+	public void testApp() {
+		assertTrue(true);
```

Comment:
It would be hard to use a live site to test and it could be fairly dynamic.
It is somewhat expensive to maintain test sites running locally within a test environment.

I would use a mock test suite like Mockito, mocking Jsoup.connect() and /or document.select() to tests different combinations.